_inc/client/plans/single-product-backup.jsx

```diff
@@ -61,89 +76,65 @@ export function SingleProductBackup( { products, upgradeLinks } ) {
 	);
 }
 
-function SingleProductBackupHeader( { currencySymbol, backupPlanPrices } ) {
-	return (
-		<div className="single-product-backup__header-container">
-			<h3 className="single-product-backup__header-title">{ __( 'Jetpack Backup' ) }</h3>
-			<PlanPriceDisplay currencySymbol={ currencySymbol } backupPlanPrices={ backupPlanPrices } />
-		</div>
-	);
-}
-
-export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
-	const dailyBackupYearlyPrice = backupPlanPrices.jetpack_backup_daily.yearly;
-	const dailyBackupMonthlyPrice = backupPlanPrices.jetpack_backup_daily.monthly;
-
-	const realtimeBackupYearlyPrice = backupPlanPrices.jetpack_backup_realtime.yearly;
-	const realtimeBackupMonthlyPrice = backupPlanPrices.jetpack_backup_realtime.monthly;
-
-	const fullDailyBackupYearlyCost = dailyBackupMonthlyPrice * 12;
-	const fullRealtimeBackupYearlyCost = realtimeBackupMonthlyPrice * 12;
-
-	const perYearPriceRange = __(
-		'%(currencySymbol)s%(dailyBackupYearlyPrice)s-%(realtimeBackupYearlyPrice)s per year',
-		{
-			args: {
-				currencySymbol,
-				dailyBackupYearlyPrice,
-				realtimeBackupYearlyPrice,
-			},
-			comment: 'Shows a range of prices, such as $12-15 per year',
-		}
-	);
+function SingleProductBackupPriceGroup( {
+	billingTimeFrame,
+	currencyCode,
+	discountedPrice,
+	fullPrice,
+} ) {
+	const isDiscounted = !! discountedPrice;
 
 	return (
-		<div className="single-product-backup__header-price">
-			<div className="discounted-price__container">
-				<div className="discounted-price__slash"></div>
-				<div className="discounted-price__price">
-					{ __( '%(currencySymbol)s%(lowPrice)s-%(highPrice)s', {
-						args: {
-							currencySymbol,
-							lowPrice: fullDailyBackupYearlyCost,
-							highPrice: fullRealtimeBackupYearlyCost,
-						},
-						comment:
-							"Describes how much a plan will cost per year. %(currencySymbol) is the currency symbol of the user's locale (e.g. $). %(planPrice) is the cost of a plan (e.g. 20).",
-					} ) }
-				</div>
-			</div>
-			<div className="plans-price__container">
-				<span className="plans-price__price-range">{ perYearPriceRange }</span>
+		<div className="single-product-backup__price-group">
+			<PlanPrice currencyCode={ currencyCode } rawPrice={ fullPrice } original={ isDiscounted } />
+			{ isDiscounted && (
+				<PlanPrice currencyCode={ currencyCode } rawPrice={ discountedPrice } discounted />
+			) }
+			<div className="single-product-backup__price-group-billing-timeframe">
```

Comment:
I don't know if that's ideal. In some languages "per year" can come before the amount.

Reply:
My reasoning here was that if there were no issues in Calypso with this approach, we should be good to re-use it here.

Reply:
If we follow the same structure in Calypso, we may want to fix it there as well, to improve localization. But I'll let you all decide.

Reply:
I'm going to change it to allow arbitrary order of price and billing time frame.

Reply:
Thanks 👍